.github/workflows/kfp-sdk-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: KFP SDK Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - 'sdk/**'
+      - '.github/workflows/kfp-sdk-tests.yml'
+
+jobs:
+  sdk-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, 3.10, 3.11, 3.12]
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+
+    - name: Install dependencies
+      run: |
```

Comment:
May you create a bash script? In this case, we can call it locally in other to have it working

Reply:
Done, I added the script back.

Reply:
Yes. Now, delegate the yaml to that script. In this case, locally and GHA will be on sync

Reply:
Done, updated to use the script here.